domains/TIMMY

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TIMMYS-UNBLOCKED-GAMES",
+        "email": "admin1223@ejmsunblocked.com"
+    },
+    "record": {
+        "CNAME": "https://timmys-unblocked-games-v3.web.app/"
```

Comment:
Invalid CNAME. CNAME records must contain no protocols, file paths, and/or URL parameters.